poppy/poppy_core.py

```diff
@@ -2059,7 +2059,7 @@ def __init__(self, name="unnamed optic", transmission=None, opd= None, opdunits=
             elif isinstance(opd, fits.HDUList):
                 # load from fits HDUList
                 self.opd_file='supplied as fits.HDUList object'
-                self.opd = opd[0].data
+                self.opd = opd[0].data.copy()
                 self.opd_header = opd[0].header
```

Comment:
I can't either, but I guess it is more consistent. When implementing this, I noticed HDUList didn't have a `copy()` method and assumed `fits.Header` didn't either, but I was mistaken. I'll add it.